casanovo/denovo/model_runner.py

```diff
@@ -105,6 +108,10 @@ def __init__(
                 filename=best_filename,
                 enable_version_counter=False,
             ),
+            LearningRateMonitor(
+                log_momentum=True,
```

Comment:
**nitpick:** I think these two can be combined in a single line without it being too wide.